measure-u1.cc

```diff
@@ -90,6 +90,15 @@ int main(int ac, char* av[]) {
       os.fill(prevf);
       os << ".dat" << std::ends;
       compute_spacial_loops(U, os.str());
+      for(size_t xp=0;xp<3; xp+=1){
```

Comment:
what is happening here? Need a function?

Reply:
I tried to test how to compute Wilson-loops for non-integer R, and wanted to check for several different combinations

Reply:
okay, but you want to leave it here like this?